src/utilities/__tests__/findBreakingChanges-test.js

```diff
@@ -1307,7 +866,7 @@ describe('findBreakingChanges', () => {
       },
       {
         type: BreakingChangeType.DIRECTIVE_REMOVED,
-        description: 'skip was removed',
+        description: 'DirectiveThatIsRemoved was removed',
```

Comment:
this was necessary since `printSchema` does not print GraphQL default directives, so I've added a directive to test the same scenario.

Reply:
Make sense 👍